### ISSUE TITLE
Feature: support comments

### ISSUE THREAD

**Issue by a0m0rajab** [open-sauced/app#2302]
### Type of feature

🍕 Feature

### Current behavior

We do not allow users to write comments on the App, it would be great if we have that.	
This would help the users to stay more on the website and have a two direction conversation in it. 

### Suggested solution

We need to implement the backend for this. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by nickytonline** [comment]
For comments, do you mean replies to highlights @a0m0rajab?

**Comment by a0m0rajab** [comment]
Yes, I mean replies to highlights.

**Comment by CBID2** [comment]
> Yes, I mean replies to highlights.

You just read my mind @a0m0rajab! :) I've been thinking about this feature for a while. It can create a more interactive experience. I think what needs to be addressed is preventing people from putting inappropriate comments underneath a person's highlights.

**Comment by a0m0rajab** [comment]
Yes, this would provide a great interaction and turn the website to let people talk together on the website this will increase the usage and even the benefits of the website for others.
